### PR TITLE
Projection dev

### PR DESCRIPTION
**Pull request by luizhdasilva**
Add option for the user to select the figure projection

### CODE REVIEW

File: anhalyze/core/anhalyze_plot_utils.py [hunk 1]
```diff
@@ -44,22 +40,91 @@ def get_feature_mask(feature='land', resolution='50m'):
...
+                 'lon_range': lon_range,
+                 'standard_parallels': standard_parallels,
+                 'central_longitude': central_longitude,
+                 'central_latitude': central_latitude,
+                 }
+
+    return proj_info
+
+
+def show_var_data_map(data, var, idepth, proj):
```
Comment (jeenriquez) [inline anhalyze/core/anhalyze_plot_utils.py:111]: Current version is crashing. Logic changed from using *kwargs* to args (i.e. `idepth=0, var=''`). Also need to change `proj` as *kwarg*

File: anhalyze/core/anhalyze_plot_utils.py [hunk 2]
```diff
@@ -44,22 +40,91 @@ def get_feature_mask(feature='land', resolution='50m'):
...
         f'[anhalyze_plot_utils] Variable {var} not found in data_vars: {list(data.data_vars)}'
 
-    # TODO better use of init_location, if location set then load it, otherwise calculate it.
-    # Init location info
-    location_info = anhalyze.core.anhalyze_geo.init_location()
+    # Calculate projection information (e.g. Standard parallels) based on the dataset lat and lon limits	
+    proj_info = get_projection_info(data)
+
+    # Select figure projection	 
+    proj_config = get_projection(proj, proj_info)
```
Comment (jeenriquez) [inline anhalyze/core/anhalyze_plot_utils.py:127]: change to *kwargs*

File: anhalyze/core/anhalyze_plot_utils.py [hunk 3]
```diff
@@ -44,22 +40,91 @@ def get_feature_mask(feature='land', resolution='50m'):
     return feature_mask
 
 
-def show_var_data_map(data, idepth=0, var=''):
+def get_projection(proj, proj_info):
```
Comment (jeenriquez) [inline anhalyze/core/anhalyze_plot_utils.py:43]: add default projection.

**Review by jeenriquez — CHANGES_REQUESTED** [review]
Hey @luizhdasilva, the code is crashing as is. If I run this, `aa.show_var_data_map('votemper')`, it crashes asking for required projection. 
I'm suggesting some changes in the code to fix this.

Please make the changes, and resubmit pull request. 
Thanks!

**Comment by luizhdasilva** [comment]
> Hey @luizhdasilva, the code is crashing as is. If I run this, `aa.show_var_data_map('votemper')`, it crashes asking for required projection. I'm suggesting some changes in the code to fix this.
> 
> Please make the changes, and resubmit pull request. Thanks!

I have done the changes. Should I close this pull request and open a new one? I am not being able to submit a pull request from the same branch because this one is open.

**Review by jeenriquez — APPROVED** [review]

**Comment by jeenriquez** [comment]
Looks good now. There are a couple of projection that may not be working well. Made an issue for it (#19). I'm not making it a requirement to merge to move forward, but should check before first release.

**Pull request merged**